docs/reference/authentication.rst

```diff
@@ -1,8 +1,9 @@
-.. _authenticate:
+.. _api-authentication:
+
+==================
+API Authentication
+==================
 
-==============
-Authentication
-==============
 
 The Flocker REST API uses TLS to secure and authenticate requests.
```

Comment:
All this feedback says to me - we don't need a glossary document for v1.0. If we're not going to use "Alpha/Beta/Production Ready" or the newly suggested "Experimental/Stable", and that the other terms aren't yet used throughout the docs, then I think this branch can be revisited post v1.

@ferrantim Please advise what you think we need for version 1.